<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWizard</class>
 <widget class="QWidget" name="StartWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start Wizard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="page_status_list_widget">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stacked_widget">
        <property name="currentIndex">
         <number>4</number>
        </property>
        <widget class="ModInstallWidget" name="module_install_widget"/>
        <widget class="FileCheckWidget" name="file_check_widget"/>
        <widget class="ParameterCheckWidget" name="parameter_check_widget"/>
        <widget class="QWidget" name="launch_widget">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="launch_label">
            <property name="font">
             <font>
              <pointsize>40</pointsize>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;All checks passed.&lt;/p&gt;&lt;p&gt;Launching Program...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="load_file_widget">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="load_file_label">
            <property name="font">
             <font>
              <pointsize>40</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Loading Files...</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>833</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ok_button">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Override</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModInstallWidget</class>
   <extends>QWidget</extends>
   <header>..modInstallWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ParameterCheckWidget</class>
   <extends>QWidget</extends>
   <header>..parameterCheckWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileCheckWidget</class>
   <extends>QWidget</extends>
   <header>..fileCheckWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
